### ISSUE TITLE
Alt Attribute

### ISSUE THREAD

**Issue by jim-rosales** [benfin28/prj-rev-bwfs-tea-cozy#1]
It's best practice to include the [alt attribute](https://en.wikipedia.org/wiki/Alt_attribute) for all the images used on a site. The reason for this is because the alt attribute optimizes your site for search engine results, increases the [accessibility](https://www.codecademy.com/articles/accessibility) of your page, and is used as a backup in case the image cannot be rendered. 

In this case, you could do something like the following:
```HTML
<img src="https://s3.amazonaws.com/codecademy-content/courses/freelance-1/unit-4/img-spiced-rum.jpg" alt="Spiced Rum Tea">
```

https://github.com/benfin28/prj-rev-bwfs-tea-cozy/blob/ac59b98a5b67bdb79e63a9123a0a7af7fe5bbd60/Tea_Cosy/index.html#L45
